test/typing/plain_files/ext/asyncio/engines.py

```diff
@@ -71,3 +72,10 @@ async def asyncio() -> None:
     ce.statement
     cc = select(1).compile(conn)
     cc.statement
+
+    async with e.connect() as conn:
+        metadata = MetaData()
+
+        await conn.run_sync(metadata.create_all)
+        await conn.run_sync(metadata.reflect)
+        await conn.run_sync(metadata.drop_all)
```

Comment:
**Michael Bayer** (zzzeek) wrote:

can we also add table.create, table.drop, index.create, index.drop, and anything else where we have an engine argument.  should be a bunch of things in schema.py

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5242

Reply:
Sounds good. I'll add these.